spec/integration/yardstick/rake/measurement_spec.rb

```diff
@@ -4,17 +4,18 @@
 require 'yardstick/rake/measurement'
 
 describe Yardstick::Rake::Measurement do
-  let(:output) { Pathname('measurements/report.txt') }
+  let(:output) { Pathname("tmp/measurements-#{Process.pid}/report.txt") }
```

Comment:
You could also use a `Tempfile` instead of this, eg:

```ruby
around do |example|
  Tempfile.new('report') do |file|
    described_class.new do |config|
      config.path   = 'lib/yardstick.rb'
      config.output = file.path
    end
    example.call
  end
end
```

A nice side benefit is that the `output.dirname.rmtree` stuff all gets removed too.